workbaskets/views/ui.py

```diff
@@ -1664,3 +1607,77 @@ def form_valid(self, form):
 
     def get_success_url(self):
         return reverse("workbaskets:current-workbasket")
+
+
+class WorkBasketCommentListView(
+    PermissionRequiredMixin,
+    WithPaginationListMixin,
+    ListView,
+):
+    permission_required = [
+        "workbaskets.view_workbasket",
+        "tasks.view_comment",
+    ]
+    template_name = "workbaskets/comments/list.jinja"
+    paginate_by = 20
+
+    @cached_property
+    def workbasket(self):
+        return WorkBasket.objects.get(pk=self.kwargs["pk"])
+
+    def get_queryset(self):
+        return Comment.objects.filter(task__workbasket=self.workbasket).order_by(
+            "-created_at",
+        )
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        context["workbasket"] = self.workbasket
+        return context
+
+
+class WorkBasketCommentUpdateDeleteMixin:
+    model = Comment
+    success_url = reverse_lazy("workbaskets:current-workbasket")
+
+    def editable(self, comment):
+        if (
+            comment.author != self.request.user
+            or comment.task.workbasket.status != WorkflowStatus.EDITING
+        ):
+            return False
+        else:
+            return True
```

Comment:
By De Morgan, could this be a more intuitive representation:
```suggestion
        return (
            comment.author == self.request.user
            and comment.task.workbasket.status == WorkflowStatus.EDITING
        )
```